### PR TITLE
Corrected links in faq and coc page

### LINKED ISSUE #26: Fix the navigation bar for FAQ and COC page.
Right now, if you go to FAQ or COC page, there are following issues:

1) The PyCon India 2018 logo is not shown in the nav bar until you scroll down.
2) `Home` part is highlighted in the nav bar.
3) None of the nav bar inks except Blog are working.

nit: both FAQ and COC should be capitalised in the nav bar.

### PR DESCRIPTION
**Pull request by akshay196**
Changes:
- Make `faq` active when on faq page. Also `coc` when on coc page 
- Corrected links in navbar of faq and coc pages
- added links for social sites in faq and coc page
Fix #26

### CODE REVIEW

File: coc.html [hunk 1]
```diff
@@ -74,15 +74,15 @@
...
+          <li><a href="index.html#journey">Journey</a></li>
           <!-- <li><a href="#speakers">Speakers</a></li> -->
           <!-- <li><a href="#conference">Schedule</a></li> -->
-          <li><a href="#sponsors">Sponsors</a></li>
+          <li><a href="index.html#sponsors">Sponsors</a></li>
           <li><a href="https://in.pycon.org/blog/">Blog</a></li>
-          <li><a href="#venue">Venue</a></li>
+          <li><a href="index.html#venue">Venue</a></li>
           <li><a href="faq.html">Faq</a></li>
-          <li><a href="coc.html">Coc  </a></li>
```
Comment (Pulkit07) [inline coc.html]: This extra space after COC should be removed in every file.

File: coc.html [hunk 2]
```diff
@@ -168,11 +168,11 @@ <h3><b>License</b></h3>
 
           <ul class="list-inline">
 
-            <li><img src="img/fb.png" alt="image"></li>
+            <li><a href="https://www.facebook.com/PyConIndia" target="_blank"><img src="img/fb.png" alt="image"></a></li>
 
-            <li><img src="img/twitt.png" alt="image"></li>
+            <li><a href="https://twitter.com/pyconindia" target="_blank"><img src="img/twitt.png" alt="image"></a></li>
 
-          <ul>
```
Comment (vnbang2003) [inline coc.html]: Can you please add mailing list button also as in index.html page?

File: faq.html [hunk 3]
```diff
@@ -219,11 +219,11 @@ <h5><b>Q. My question is not listed here / I am still unclear about a few things
 
           <ul class="list-inline">
 
-            <li><img src="img/fb.png" alt="image"></li>
+            <li><a href="https://www.facebook.com/PyConIndia" target="_blank"><img src="img/fb.png" alt="image"></a></li>
 
-            <li><img src="img/twitt.png" alt="image"></li>
+            <li><a href="https://twitter.com/pyconindia" target="_blank"><img src="img/twitt.png" alt="image"></a></li>
 
-          <ul>
```
Comment (vnbang2003) [inline faq.html]: Can you please add mailing list button as in index.html page?

Comment (akshay196) [reply faq.html]: done.

File: coc.html (hunk 2 above)
Comment (akshay196) [reply coc.html]: done

File: sponsorship.html [hunk 4]
```diff
@@ -106,9 +106,9 @@
 
           <li><a href="index.html">Venue</a></li>
```
Comment (SaptakS) [inline sponsorship.html]: It would be great if you could correct link on this page also. Thanks.

**Pull request merged**